src/zenml/models/v2/base/scoped.py

```diff
@@ -245,6 +245,8 @@ def apply_sorting(
                 UserSchema, getattr(table, "user_id") == UserSchema.id
             )
 
+            query = query.add_columns(UserSchema.name)
```

Comment:
Can you explain why this is necessary?

From what I read, this essentially means the query is now a
`select(original_query.output, user_name)` query? This would be very confusing IMO, that people using the filter model now suddenly get extra items in their query results

Reply:
You may have seen this pattern in different places in this PR. This is needed because eventually we need to order by this column and if we don't add this here, the entire thing fails with an `SQLOperationalError`, simply due to the fact that it can not find the column in the initial selection.

I realize this might be confusing at first glance, but as you can see from the code, I did not have to make any adjustments to the Schema -> Response conversion for the corresponding Schemas. This means that our developers won't have to manage it and also, since this is entirely isolated to our internal code, our users will not be affected by it either.